roles/draupnir/handlers/main.yml

```
@@ -0,0 +1,22 @@
+---
+
+- name: Config bot draupnir service
+  ansible.builtin.systemd:
+    daemon_reload: true
+    name: matrix-bot-draupnir.service
+    enabled: true
+
+- name: Draupnir build
+  become: true
+  become_user: draupnir
+  ansible.builtin.shell: |
+    cd {{ draupnir_data_path }}/src
+    yarn build
```

tässä ei liene tarvetta shellille?
```suggestion
  ansible.builtin.command:
    cmd: yarn build
    chdir: "{{ draupnir_data_path }}/src"
```